superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -288,11 +288,20 @@ export default function transformProps(
         entry,
         ensureIsArray(chartProps.rawFormData?.time_compare),
       )!;
-      if (!offsetLineWidths[offset]) {
-        offsetLineWidths[offset] = Object.keys(offsetLineWidths).length + 1;
+      if (!offsetLineWidths.includes(offset)) {
```

Comment:
The map approach is more efficient than looping through the elements with `include`.